doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
+- Author(s): Emmanuel Baccelli  
+- Date: December 2016
+
+
+RIOT Developer Memo Format, Publishing and Maintenance Process
```

Comment:
Looking at the proposed rdm again, I would also suggest to move the title to the top, followed by author (with contact information) and date iff need be. I would really leave this to git as it is an item that will likely be overlooked on a change anyways.